array.rb

```diff
@@ -73,35 +73,52 @@ def shuffle(random: Random)
   end
 
   # call-seq:
-  #    array.sample(random: Random) -> object
-  #    array.sample(n, random: Random) -> new_ary
+  #   sample(random: Random.new) -> object
+  #   sample(count, random: Random.new) -> new_ary
```

Comment:
The default value is the `Random` class, not an instance of `Random`. The `Random` class implements the random methods as instance methods.

```suggestion
  #   sample(random: Random) -> object
  #   sample(count, random: Random) -> new_ary
```